server/src/types/type.py

```diff
@@ -0,0 +1,50 @@
+from datetime import date
+
+class User:
+    def __init__(self, fname=None, lname=None, phone=None, address=None):
+        if fname is None:
+            raise ValueError('ValueError: First name not specified')
+        if lname is None:
+            raise ValueError('ValueError: Last name not specified')
+        if phone is None:
+            raise ValueError('ValueError: Phone number not specified')
+        if address is None:
+            raise ValueError('ValueError: Address not specified')
+
+        self.fname = fname.strip().lower()
+        self.lname = lname.strip().lower()
+        self.phone = phone
+        self.posts = {}
+
+        """
+            address format: building,city,state,zip
+                i.e. 370 Jay Street,Brooklyn,NY,11201
+        """
+        address = address.lower().split()
```

Comment:
This is fine for now, but I think later on we would want a centralized storage of all addresses in the system. This way we don't show 2 different lists when the address is off by like a whitespace or a comma.

Reply:
yessir